lib/v2/snow-peak/new-arrivals.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const cheerio = require('cheerio');
+const host = 'https://www.snowpeak.com';
+module.exports = async (ctx) => {
+    const url = `${host}/collections/new-arrivals`;
+    const response = await got({
+        method: 'get',
+        url,
+    });
+    const data = response.data;
+
+    const $ = cheerio.load(data);
+    const list = $('.element.product-tile')
+        .map(function () {
+            const data = {};
+            const details = $(this).find('.product-data').eq(0).attr('data-product');
+            const product = JSON.parse(details);
```

Comment:
```suggestion
            const product = $(this).find('.product-data').data('product');
```
You can use [.data()](https://cheerio.js.org/docs/api/classes/Cheerio#data)